src/js/actions/layers.js

```diff
@@ -714,13 +714,16 @@ define(function (require, exports) {
                     }
                 }).then(function () {
                     return this.transfer(tools.resetBorderPolicies);
+                }).then(function () {
+                    return this.transfer(tools.resetPenTool);
```

Comment:
Does this need to wait for `resetBorderPolicies`? Or could these both be `join`d together?